packages/yoga/src/Tag/Tag.theme.js

```diff
@@ -1,21 +1,24 @@
-const Tag = ({ spacing, fontSizes, fontWeights, radii }) => ({
+const Tag = ({ spacing, fontSizes, fontWeights, radii, borders }) => ({
   icon: {
+    size: spacing.small,
     margin: {
       right: spacing.xxxsmall,
     },
   },
   font: {
     size: fontSizes.xsmall,
-    weight: fontWeights.bold,
+    lineHeight: fontSizes.medium,
+    weight: fontWeights.medium,
   },
   padding: {
-    top: spacing.xxsmall,
+    top: spacing.xxxsmall,
     right: spacing.xxsmall,
-    bottom: spacing.xxsmall,
+    bottom: spacing.xxxsmall,
     left: spacing.xxsmall,
   },
   border: {
-    radius: radii.rounded,
+    width: borders.small,
```

Comment:
@andrerocha22 Are we using this `width` attribute?

Reply:
We're using here: `border: ${tag.border.width}px solid`

Reply:
since `border-style` is solid by default, we should change only what is needed to change. So, we should modify `border-width` instead.